app/config/LocalTemplateRenderer.scala

```diff
@@ -38,7 +37,7 @@ class LocalTemplateRenderer @Inject() (
   override lazy val refreshAfter: Duration =
     runModeConfiguration.getOptional[Int]("template.refreshInterval").getOrElse(600) seconds
 
-  private implicit val hc = HeaderCarrier()
+  private implicit val hc = HeaderCarrier(sessionId = Some(SessionId("sessionId-00000-00000-00000")))
```

Comment:
This should not be here. If it is for a test it should be somewhere else. Or is it some debugging tests that should be deleted?

Reply:
Done